src/app/(auth)/login/page.tsx

```diff
@@ -65,32 +65,34 @@ export default function Login() {
             value={email}
             setValue={setEmail}
           />
-          <TextInput
-            label="Password"
-            placeholder="Password"
-            errorText={passwordError}
-            type="password"
-            id="password"
-            value={password}
-            setValue={setPassword}
-          />
+          <SpacerDiv $gap={0.813}>
```

Comment:
I don't think the SpacerDiv here is necessary? (the only child component is the TextInput, so the gap isn't doing anything here either)